include/baresip.h

```diff
@@ -216,6 +216,7 @@ typedef void (call_list_h)(struct call *call, void *arg);
 
 int  call_connect(struct call *call, const struct pl *paddr);
 int  call_answer(struct call *call, uint16_t scode, enum vidmode vmode);
+int  call_ringing(const struct call *call);
```

Comment:
If we add an API function now, it will be difficult to remove it later after https://github.com/baresip/baresip/pull/3090 an following bevent PRs are done.